static/app/views/projects/gettingStartedWithProjectContext.tsx

```diff
@@ -0,0 +1,49 @@
+import {createContext} from 'react';
+
+import {OnboardingSelectedSDK, Project} from 'sentry/types';
```

Comment:
new context used to save project data. This data is later displayed as default values project creation form if users choose to go back in the flow